pkg/cmd/version/version.go

```diff
@@ -15,22 +15,92 @@
 package version
 
 import (
+	"encoding/json"
 	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+	"time"
 
+	"github.com/blang/semver"
+	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 )
 
 // NOTE: use go build -ldflags "-X github.com/tektoncd/cli/pkg/cmd/version.clientVersion=$(git describe)"
 var clientVersion = "dev"
 
+const latestReleaseURL = "https://api.github.com/repos/tektoncd/cli/releases/latest"
+
 // Command returns version command
 func Command() *cobra.Command {
+	var check bool
+
 	var cmd = &cobra.Command{
 		Use:   "version",
 		Short: "Prints version information",
 		Run: func(cmd *cobra.Command, args []string) {
 			fmt.Fprintf(cmd.OutOrStdout(), "Client version: %s\n", clientVersion)
+
+			if check {
+				current, latest, err := getVersions()
+				if err != nil {
+					fmt.Fprintf(cmd.OutOrStderr(), "Failed to get the latest version: %s\n", err)
+					return
+				}
+
+				if current.LT(*latest) {
+					fmt.Fprintf(cmd.OutOrStdout(), "The latest version v%s is avairable. Please update.\n", latest)
```

Comment:
perhaps we can print the `html_url` field so the user can easily go to the webpage of the version ? i.e: 

<img width="862" alt="image" src="https://user-images.githubusercontent.com/98980/61027079-83f83280-a3b5-11e9-8bf5-07b315a2d426.png">